actors/evm/src/lib.rs

```diff
@@ -252,7 +287,9 @@ impl ActorCode for EvmContractActor {
         // We reserve all methods below EVM_MAX_RESERVED (<= 1023) method. This is a _subset_ of
         // those reserved by FRC0042.
         if method > EVM_MAX_RESERVED_METHOD {
-            return Self::invoke_contract(rt, method, params, false, None).map(RawBytes::new);
+            // FIXME: we need the actual codec.
```

Comment:
For my understanding, what is the codec communicating?

Reply:
The parameter codec.